docs/data/migration/migration-data-grid-v6/migration-data-grid-v6.md

```diff
@@ -106,9 +106,10 @@ As a result, the following changes have been made:
 - Pinned row and column sections are now contained in the virtual scroller.
 - The cell inner wrapper `.MuiDataGrid-cellContent` has been removed.
 
-<!-- ### Renamed props
+## Renamed props
 
-- -->
+- The props `rowBuffer`, `columnBuffer`, `rowThreshold` and `columnThreshold` were renamed to `rowBufferPx`, `columnBufferPx`, `rowThresholdPx` and `columnThresholdPx`.
```

Comment:
Nit: We might also call it the removal of some props and addition of some to make things more explicit. Since the API and expected values are updated.

Reply:
I think the rename entry with the explanation is fine. The 2 removed props have been moved to their section.